lib/pluginmanager/util.rb

```diff
@@ -1,12 +1,35 @@
 require "rubygems/package"
 
 module LogStash::PluginManager
+
+  class Settings
```

Comment:
I agree with the `LogStash::PluginManager` vs `util.rb` confusion, opened issue #3746 for that.